static/script/widgets/grid.js

```diff
@@ -24,18 +24,24 @@ define(
          * @param {String} [id] The unique ID of the widget. If excluded, a temporary internal ID will be used (but not included in any output).
          * @param {Integer} cols The number of columns in the grid.
          * @param {Integer} rows The number of rows in the grid.
+         * @param {boolean} horizontal_wrapping Enable or disable horizontal wrapping.
+         * @param {boolean} vertical_wrapping Enable or disable vertical wrapping.
          */
         var Grid = Container.extend(/** @lends antie.widgets.Grid.prototype */ {
             /**
              * @constructor
              * @ignore
              */
-            init: function (id, cols, rows) {
+            init: function (id, cols, rows, horizontal_wrapping, vertical_wrapping) {
```

Comment:
Can we change these variables to camelCase please (remember to change the JSDoc too)?